x-pack/plugins/ml/public/application/routing/routes/timeseriesexplorer/state_manager.tsx

```diff
@@ -257,7 +258,7 @@ export const TimeSeriesExplorerUrlStateManager: FC<TimeSeriesExplorerUrlStateMan
         });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [selectedForecastId]);
+  }, [selectedForecastId, selectedForecastId]);
```

Comment:
`selectedForecastId` is repeated here. it didn't flag up because we're suppressing the `exhaustive-deps` error :(

Reply:
Removed in 0d7a3abd9109e4798d648f244ed6b56fe2c3d10b